problem_builder/mentoring.py

```diff
@@ -762,7 +771,7 @@ def student_view_data(self, context=None):
         for child_id in self.children:
             block = self.runtime.get_block(child_id)
             if hasattr(block, 'student_view_data'):
-                components.append(block.student_view_data())
+                components.append({block.category: block.student_view_data()})
```

Comment:
Instead of changing the shape of the data to `{block.category: block.student_view_data()}`, could we instead do this:

```python
{
  'category': block.category',
  'student_view_data': block.student_view_data()
}
```

This will allow us to put additional metadata to the components array in the future - child block id for example would be a good candidate.